drivers/SmartThings/zwave-switch/src/test/test_zooz_zen_30_dimmer_relay.lua

```diff
@@ -1026,5 +1040,21 @@ test.register_message_test(
   }
 )
 
+test.register_coroutine_test(
+  "PROFILE CHANGE",
+  function()
+    test.timer.__create_and_queue_test_time_advance_timer(1, "oneshot")
+    test.socket.zwave:__queue_receive({
+      mock_zooz_zen_30_dimmer_relay.id,
+      Version:Report(
+        {
+          firmware_0_version = 1,
+          firmware_0_sub_version = 5
+        }
```

Comment:
@greens here I have a problem with testing change of version. There are strange logs while I report version with firmware 1.05:
Received unexpected error: ...op\SmartThingsEdgeDriversFork\lua_libs-api_v0\cosock.lua:303: ...\SmartThingsEdgeDriversFork\lua_libs-api_v0\st\utils.lua:188: bad argument #2 to 'pack' (number expected, got nil).
Could you please take a look on it?